invenio_communities/assets/semantic-ui/js/invenio_communities/settings/privileges/CommunityPrivilegesForm.js

```diff
@@ -124,15 +124,44 @@ MemberPolicyField.defaultProps = {
   label: "",
 };
 
+const SubmissionPolicyField = ({ label, formConfig, ...props }) => {
+  const [field] = useField(props);
+  return (
+    <>
+      {formConfig.access.submission_policy.map((item) => (
```

Comment:
minor:
```suggestion
  const [field] = useField(props);
  const {submission_policy: subPolicy} = formConfig.access;
  return (
    <>
      {formConfig.access.submission_policy.map((item) => (
```
for better readability in the map function